src/components/Filter/FilterContent.tsx

```diff
@@ -228,6 +248,9 @@ const FilterContent: React.FC<FilterContentProps> = ({
                         {filterField.multiple ? (
                           <>
                             <TextField
+                              data-test={filterFieldTestingContext}
+                              data-test-id={filterField.name}
+                              data-test-range-type="min"
```

Comment:
What does it mean `data-test-range-type` and why having different `data-test` is not enough? I thought till now, that we wanted to have only `data-test` and `data-test-id`.

Reply:
It's better to put extra `data-test-*` attribute rather than miss one 🤷‍♂️